src/javascript/app_2/App/Components/Form/DatePicker/date_picker.jsx

```diff
@@ -6,7 +6,7 @@ import {
     IconArrow,
     IconCalendar,
     IconClear }             from 'Assets/Common';
-import InputField           from 'App/Components/Form/input_field.jsx';
+import InputField           from 'App/Components/Form/InputField/input_field.jsx';
```

Comment:
since this is moved to a folder of its own, i suggest to add an`index.js` file in InputField folder that exports `input_field.jsx` component, then we can simply import this component as such: `App/Components/Form/InputField`

Reply:
correct, thanks for the suggestion :+1: